src/library/scala/collection/TraversableOnce.scala

```diff
@@ -351,9 +351,7 @@ trait TraversableOnce[+A] extends Any with GenTraversableOnce[A] {
 
   def toMap[T, U](implicit ev: A <:< (T, U)): immutable.Map[T, U] = {
     val b = immutable.Map.newBuilder[T, U]
-    for (x <- self)
-      b += x
-
+    b ++= seq
```

Comment:
This would need to be `seq.asInstanceOf[TraversableOnce[(T, U)]`

Reply:
doh - fixed now